### PR TITLE
Added vehicle position-heading

### PR DESCRIPTION
**Pull request by EddyK69**
## Type of change
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (which adds functionality to this library)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue: 

## Checklist

- [X] The code change is tested and works locally.
- [ ] Tests have been added to verify that the new code works.

### CODE REVIEW

File: bimmer_connected/vehicle_status.py [hunk 1]
```diff
@@ -143,6 +143,20 @@ def gps_position(self) -> (float, float):
...
+    def gps_heading(self) -> (int):
+        """Get the last known heading of the vehicle.
+
+        This only provides data, if the vehicle tracking is enabled!
+        """
+        pos = self._state.attributes[SERVICE_STATUS]['position']
+        if not self.is_vehicle_tracking_enabled:
+            _LOGGER.warning('Positioning status is %s', pos['status'])
+            return None
+        pos = self._state.attributes[SERVICE_STATUS]['position']
```
Comment (gerard33) [inline bimmer_connected/vehicle_status.py:157]: This line is already in 153. I see the same goes for lines 143 and 139.

**Comment by gerard33** [comment]
The heading of the car is something nice to add indeed. I used it earlier on the map in HA to show how the car was positioned (not using that anymore now). We can add this in HA as attribute of the device_tracker.

Can you also add some tests for this?

**Comment by EddyK69** [comment]
Yep, already did that (both tests & device_tracker attribute in HA) 👍

File: bimmer_connected/vehicle_status.py (hunk 1 above)
Comment (EddyK69) [reply bimmer_connected/vehicle_status.py:155]: True, but refactoring doesn’t makes it easier to read though. Did some minot changes

**Review by gerard33 — APPROVED** [review]

**Pull request merged**